src/testdir/test_visual.vim

```diff
@@ -806,11 +806,8 @@ func Test_visual_block_mode()
   %d _
   call setline(1, ['aaa', 'bbb', 'ccc'])
   exe "normal $\<C-V>2jA\<Left>x"
-  " BUG: Instead of adding x as the third character in all the three lines,
-  " 'a' is added in the second and third lines at the end. This bug is not
-  " reproducible if this operation is performed manually.
-  "call assert_equal(['aaxa', 'bbxb', 'ccxc'], getline(1, '$'))
-  call assert_equal(['aaxa', 'bbba', 'ccca'], getline(1, '$'))
+  call assert_equal(['aaxa', 'bbxb', 'ccxc'], getline(1, '$'))
+  "call assert_equal(['aaxa', 'bbba', 'ccca'], getline(1, '$'))
```

Comment:
You uncommentted out the first `assert_equal` (which seems good) but you commented out the 2nd one.
Should the 2nd assert really be commented out?  If yes, we probably need  a comment to say why.  It's not clear to me at least but I did not spend much time looking at the change.

Reply:
Yes, that commented second one is of course wrong and should be removed

Reply:
ah, Bram had already fixed it.